### PR TITLE
feat: extra_configs for istio-ingress & custom release-name for all addons.

### PR DESCRIPTION
**Pull request by h1manshu98**
- Added feature to provide extra helm_release attributes in istio-ingress.
- Added a new Addon `FluentBit` which by default collects logs from Pods running on EKS cluster and sends them to CloudWatch LogGroup.
- Removed `config` directory (which contains override-value.yaml) from `basic` directory as it not required there.

### CODE REVIEW

File: _examples/complete/config/istio/virtual-service.yaml [hunk 1]
```diff
@@ -7,16 +7,16 @@ kind: VirtualService
...
     meta.helm.sh/release-name: appname
-    meta.helm.sh/release-namespace: istio-system
+    meta.helm.sh/release-namespace: default
   generation: 1
   labels:
     app.kubernetes.io/instance: appname
     app.kubernetes.io/name: appname
   name: appname
-  namespace: istio-system
+  namespace: default
```
Comment (anmolnagpal) [inline _examples/complete/config/istio/virtual-service.yaml:16]: why default?

File: _examples/complete/config/override-fluent-bit.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  requests:
+    cpu: 100m
+    memory: 90Mi
+
+podAnnotations:
+  co.elastic.logs/enabled: "true"
+
+
+# -- EKS Cluster Details
+# eks_configs:
```
Comment (anmolnagpal) [inline _examples/complete/config/override-fluent-bit.yaml:27]: can we remove this if not needed

File: _examples/complete/main.tf [hunk 3]
```diff
@@ -194,55 +184,58 @@ module "addons" {
...
+  # kiali_server              = true
+  # kiali_manifests           = var.kiali_manifests
+  # external_secrets          = true
+  # externalsecrets_manifests = var.externalsecrets_manifests
+
+  # # -- Path of override-values.yaml file
+  # metrics_server_helm_config               = { values = ["${file("./config/override-metrics-server.yaml")}"] }
+  # cluster_autoscaler_helm_config           = { values = ["${file("./config/override-cluster-autoscaler.yaml")}"] }
+  # karpenter_helm_config                    = { values = ["${file("./config/override-karpenter.yaml")}"] }
+  # aws_load_balancer_controller_helm_config = { values = ["${file("./config/override-aws-load-balancer-controller.yaml")}"] }
```
Comment (anmolnagpal) [inline _examples/complete/main.tf:211]: if not needed please remove them

**Review by anmolnagpal — CHANGES_REQUESTED** [review]

File: _examples/complete/config/istio/virtual-service.yaml (hunk 1 above)
Comment (h1manshu98) [reply _examples/complete/config/istio/virtual-service.yaml:16]: its an example of namespace, user can use their namespace name in place of default, there is no depenncies of virtual-service with istio-ingress addon.

File: _examples/complete/config/override-fluent-bit.yaml (hunk 2 above)
Comment (h1manshu98) [reply _examples/complete/config/override-fluent-bit.yaml:27]: yes, we can. I have added these details inside `addons/fluent-bit/main.tf` using `set_values`

File: _examples/complete/main.tf (hunk 3 above)
Comment (h1manshu98) [reply _examples/complete/main.tf:211]: commented by mistake, we need these attributes.

**Review by VishwajitNagulkar — APPROVED** [review]
lgtm

**Review by omsharma07 — APPROVED** [review]
LGTM

**Review by nikitadugar — APPROVED** [review]
!LGTM

**Review by AshutoshAM2002 — APPROVED** [review]

**Pull request merged**